content/docs/studio/user-guide/teams.md

```diff
@@ -112,30 +113,57 @@ To add a view to this dashboard, click on `Add a view`. The process for adding a
 view is the same as that for adding personal views
 ([instructions](/doc/studio/user-guide/views/create-view)).
 
-### Settings
+## Settings
 
 In the team settings page, you can change the team name, add credentials for the
 data remotes, and delete the team. Note that these settings are applicable to
 the team and are thus different from
 [view settings](/doc/studio/user-guide/views/view-settings).
 
 Additionally, you can also
-[manage connections to self-hosted GitLab servers](#manage-connections-to-self-hosted-gitlab-servers)
-and [edit collaborators](#edit-collaborators).
+[manage connections to self-hosted GitLab servers](#manage-connections-to-self-hosted-gitlab-servers),
+[edit collaborators](#edit-collaborators) and
+[change your team plan](#change-your-team-plan).
 
-#### Manage connections to self-hosted GitLab servers
+### Manage connections to self-hosted GitLab servers
 
 If your team’s Git repositories are on a self-hosted GitLab server, you can go
 to the `GitLab connections` section of the team settings page to set up a
 connection to this server. Once you set up the connection, all your team members
 can connect to the Git repositories on this server. For more details, refer to
 [Custom GitLab Server Connection](/doc/studio/user-guide/connect-custom-gitlab-server).
 
-#### Edit collaborators
+### Edit collaborators
 
 To manage the collaborators (team members) of your team, go to the
 `Collaborators` section of the team settings page. Here you can invite new team
 members as well as remove or change the [roles](#roles) of existing team
 members.
 
-![](https://static.iterative.ai/img/studio/team_settings_v3.png)
+The number of collaborators in your team depends on your team plan. By default,
+all teams are on the Free plan, and can have 2 collaborators. To add more
+collaborators, [upgrade to the Team or Enterprise plan](#change-your-team-plan).
+
+All collaborators and pending invites get counted in the subscription. Suppose
+you have subscribed for a 10 member team. If you have 5 members who have
```

Comment:
```suggestion
you have subscribed for a 10 members team. If you have 5 members who have
```
?

Reply:
From what I know, it should be **10 member team** or **10-member team**, without `s`. So, not changing this. Let me know if you disagree.